services/proxy/pkg/config/config.go

```diff
@@ -84,6 +85,12 @@ type AuthMiddleware struct {
 	CredentialsByUserAgent map[string]string `yaml:"credentials_by_user_agent"`
 }
 
+// PoliciesMiddleware configures the proxy policies middleware.
+type PoliciesMiddleware struct {
+	Enabled bool   `yaml:"enabled" env:"PROXY_POLICIES_ENABLED" desc:"Activates policy evaluation."`
+	Query   string `yaml:"query" env:"PROXY_POLICIES_QUERY" desc:"Defines the policy query."`
```

Comment:
```suggestion
	Query   string `yaml:"query" env:"PROXY_POLICIES_QUERY" desc:"Defines the 'Complete Rules' variable defined in the rego rule set this step uses for its evaluation. Defaults to deny if the variable was not found."`
```
The naming is derived from: https://www.openpolicyagent.org/docs/latest/#complete-rules
`Pls check that if there is no match, the evaluation defaults to deny!`
Add a log entry if no match was found so that admins can check if they mistyped the name.